src/clr-angular/button/button-loading/loading-button.spec.ts

```diff
@@ -33,15 +33,15 @@ describe('Loading Buttons', () => {
   });
 
   it('displays spinner when [(clrButtonState)] value is LOADING', () => {
-    fixture.componentInstance.buttonState = ClrLoadingState.LOADING;
+    fixture.componentInstance.buttonState = <ClrLoadingState>ClrLoadingState.LOADING;
```

Comment:
This is probably only needed because our unit tests typing is a bit shaky. But it really doesn't matter for this upgrade.

Reply:
It was due to TS version bump. It was expecting the `buttonState` to be typed as `ClrLoadingState.DEFAULT`.